packer/vault.json.example

```diff
@@ -14,46 +14,58 @@
     {
       "name": "vault",
       "type": "amazon-ebs",
+      "communicator": "ssh",
+      "ssh_username": "ec2-user",
       "region": "{{ user `builder_region` }}",
       "source_ami_filter": {
         "filters": {
-          "name": "CentOS Linux 7 x86_64 HVM EBS *"
+          "name": "amzn2-ami-hvm-*-x86_64-gp2",
```

Comment:
Hey @lingrino - If you must, would you consider making the ami name and owner a variable?